libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Spec/Conway.hs

```diff
@@ -40,3 +31,31 @@ spec = describe "Conway conformance tests" $ do
     genEnv `generatesWithin` 3_000_000
     genSt `generatesWithin` 40_000_000
     genSig `generatesWithin` 60_000_000
+  describe "Conformance" $ do
```

Comment:
Not sure about expressing this taxonomy here - I think it's useful to have it as a model, for example to split the work into tasks, but I don't know if it brings anything here . The "levels" are maybe understandable for us, but it's not necessarily obvious what they mean, nor do they bring useful information. 
I would keep this flat as it was .

Reply:
I just wanted this file to correspond to the issue structure so it is easy to confirm that the issues resolved are indeed working tests in the code. And the plan was to flatten the structure after all coverage is done, since then "level" and such makes very little sense. I will flatten these a bit more right now, according to your suggestion. :+1: so I keep the order the same but collapse the superfluous hierarchy. Thanks! :raised_hands:

Reply:
I have retained the `ticks` vs `blocks` graph nesting so that it is clear that, for example `DELEG` does not depend on `NEWEPOCH`.